tests/analog/message.test.ts

```diff
@@ -29,7 +29,210 @@ const downlinkMessages: TMessageExamples = {
             actual: 0xfe
         }
     },
-    'invalid correctTime2000': {
+    'setParameters + getParameters': {
+        bytes: getBytesFromHex('03 02 04 0c 03 0b 1a 52 b8 09 42 52 b8 2d 42 17 00 04 01 04 04 01 1a 63'),
+        commands: [
+            {
+                id: downlinkCommands.setParameter.id,
+                name: 'setParameter',
+                headerSize: 2,
+                parameters: {id: 4, data: {value: 12}},
+                bytes: getBytesFromHex('03 02 04 0c')
+            },
+            {
+                id: downlinkCommands.setParameter.id,
+                name: 'setParameter',
+                headerSize: 2,
+                parameters: {
+                    id: 26,
+                    data: {
+                        latitude: 34.43,
+                        longitude: 43.43,
+                        altitude: 23
+                    }
+                },
+                bytes: getBytesFromHex('03 0b 1a 52 b8 09 42 52 b8 2d 42 17 00')
+            },
+            {
+                id: downlinkCommands.getParameter.id,
+                name: 'getParameter',
+                headerSize: 2,
+                parameters: {id: 4, data: null},
+                bytes: getBytesFromHex('04 01 04')
+            },
+            {
+                id: downlinkCommands.getParameter.id,
+                name: 'getParameter',
+                headerSize: 2,
+                parameters: {id: 26, data: null},
+                bytes: getBytesFromHex('04 01 1a')
+            }
+        ],
+        lrc: {
+            expected: 0x63,
+            actual: 0x63
+        }
+    },
+    'multiple commands in the message': {
+        bytes: getBytesFromHex(
+            '1f 30 05 2f 97 0c 02 01 1f 0d 04 2f 98 01 01 1f 0c 04 2f 97 0c 01 1f 0c 04 2f 97 0c 01 1f 05 00 1f 02 00 1b'
```

Comment:
you can use string templates here

it's possible to split in this way: one line - one command

Reply:
reworked to string templates